src/details/ArborX_DetailsHalfTraversal.hpp

```diff
@@ -54,8 +55,7 @@ struct HalfTraversal
   {
     auto const predicate =
         _get_predicate(HappyTreeFriends::getLeafBoundingVolume(_bvh, i));
-    auto const leaf_permutation_i =
-        HappyTreeFriends::getLeafPermutationIndex(_bvh, i);
+    auto const leaf_permutation_i = HappyTreeFriends::getValue(_bvh, i).index;
```

Comment:
Yuk.  I don't have a better idea though.

Reply:
I agree, would be nice to just have `getValue()` here. If we ever figure out how to move away from storing `(index, volume)` together in the future, this would be possible. But right now, it comes with severe penalties.